core/identity-management/iin-agent/README.md

```diff
@@ -33,3 +33,43 @@ make build
 ## Running the IIN Agent
 
 Run `npm run dev` to start the agent in your terminal. (_Note_: we will support a containerized IIN agent in the future.)
+
+## Configuration
+
+### Security Domain Config
+
+Sample `security-domain-config.json`:
+```
+{
+    "securityDomainName": "<ledgerId>",
+    "securityDomainName": [
+        "<node1/org1>",
+        "<node2/org2>"
+    ]
+}
+```
+
+It is JSON, with specifying who are the members of a security domain, with keys being the security domain.
+* For Fabric, value will be the channel name.
+* For Corda, value will be an array of nodes' endpoints. (Not implemented yet)
+
+### DNS Config
+
+Discovery is not implemented yet, so DNS details are added during bootstrapping. A sample `dnsconfig.json` looks like this:
+```
+{
+    "securityDomainName": {
+        "iin-agent1-name": {
```

Comment:
I think this is also not a keyword but something to be replaced with a real name, right?